python/CMakeLists.txt

```diff
@@ -81,12 +81,7 @@ file(COPY "${PROJECT_SOURCE_DIR}/VERSION" DESTINATION "${python_mod_path}")
 # Set the installation path
 
 # Ask Python where it keeps its system (platform) packages.
-
-if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-    file(WRITE "${CMAKE_BINARY_DIR}/install-prefix" "")
-else()
-    file(WRITE "${CMAKE_BINARY_DIR}/install-prefix" "${CMAKE_INSTALL_PREFIX}")
-endif()
+file(WRITE "${CMAKE_BINARY_DIR}/install-prefix" "${CMAKE_INSTALL_PREFIX}")
 
 execute_process(
     COMMAND ${PYTHON_EXECUTABLE} -c
```

Comment:
This still sets the python package destination by default to what sysconfig says, not `${CMAKE_INSTALL_PREFIX}/lib/python3.9/site-packages`

Reply:
Actually, it sets it to `${CMAKE_INSTALL_PREFIX}/lib/python3.9/site-packages`.
The contents of `${CMAKE_BINARY_DIR}/install-prefix` are set to `CMAKE_INSTALL_PREFIX`.
That is passed into the Python script, which will append the `/lib/python3.x/site-packages`.
I just double checked, and it works.
Try yourself by hard-coding a non empty value for prefix:
```
 > python -c "import sys,sysconfig;pfx='/home/brent/fancy-neuro-env';print(sysconfig.get_path('platlib',vars={} if pfx=='' else {'base':pfx,'platbase':pfx}))"
/home/brent/fancy-neuro-env/lib/python3.9/site-packages
```

Reply:
Right!